src/controller/tests/users.controller.test.ts

```diff
@@ -0,0 +1,195 @@
+import Testing from '../../utils/testing.utils'
+
+import controller from '../users.controller'
+
+import UserService from '../../services/user.service'
+
+import { GenericResponse, NotFound, Updated } from '../../utils/apiResponse.utils'
+import User from '../../model/users.model'
+
+// Testing Globals
+let req
+let res
+let next
+
+let expectedResults
+let expectedResult
+let expectedError
+
+describe('UserController', () => {
+  beforeEach(() => {
+    req = Testing.fakeRequest()
+    res = Testing.fakeResponse()
+    next = Testing.fakeNext()
+
+    expectedResults = []
```

Comment:
I left this empty because the content doesn't really need to be checked so much as that the actual list is being returned. If we added users to this list, the tests wouldn't need to change as it'd be checking the list itself regardless.

That said, we certainly could add users here for a more accurate representation of what will be happening in production.

Iirc I remember using a library called `faker` (or something like that) a bit back that allows easily fake data (though that may have been not in node 😬)

I'll look into it and see if we can't get something like that added here as I think it'd be useful in the future for other tests (and we can use it here too)